resources/ts/Components/IntegrationCard.tsx

```diff
@@ -29,7 +29,7 @@ type Props = Integration &
   };
 
 const productTypeToPath = {
-  "uitpas-api": "/",
+  uitpas: "/uitpas/getting-started",
```

Comment:
Shouldn't this be `uitpas-api`?
```suggestion
  "uitpas-api": "/uitpas/getting-started",
```

Reply:
Since it's uitpas in the backend, I have also changed it to uitpas in the frontend.